README.md

```diff
@@ -37,114 +37,16 @@ Tekton Pipelines are **Typed**:
-| [v0.6.0](https://github.com/tektoncd/pipeline/releases/tag/v0.6.0)           | [Docs @ v0.6.0](https://github.com/tektoncd/pipeline/tree/release-v0.6.x/docs#tekton-pipelines)   | [Examples @ v0.6.0](https://github.com/tektoncd/pipeline/tree/v0.6.0/examples#examples)           |
-| [v0.5.2](https://github.com/tektoncd/pipeline/releases/tag/v0.5.2)           | [Docs @ v0.5.2](https://github.com/tektoncd/pipeline/tree/v0.5.2/docs#tekton-pipelines)           | [Examples @ v0.5.2](https://github.com/tektoncd/pipeline/tree/v0.5.2/examples#examples)           |
-| [v0.5.1](https://github.com/tektoncd/pipeline/releases/tag/v0.5.1)           | [Docs @ v0.5.1](https://github.com/tektoncd/pipeline/tree/v0.5.1/docs#tekton-pipelines)           | [Examples @ v0.5.1](https://github.com/tektoncd/pipeline/tree/v0.5.1/examples#examples)           |
-| [v0.5.0](https://github.com/tektoncd/pipeline/releases/tag/v0.5.0)           | [Docs @ v0.5.0](https://github.com/tektoncd/pipeline/tree/v0.5.0/docs#tekton-pipelines)           | [Examples @ v0.5.0](https://github.com/tektoncd/pipeline/tree/v0.5.0/examples#examples)           |
-| [v0.4.0](https://github.com/tektoncd/pipeline/releases/tag/v0.4.0)           | [Docs @ v0.4.0](https://github.com/tektoncd/pipeline/tree/v0.4.0/docs#tekton-pipelines)           | [Examples @ v0.4.0](https://github.com/tektoncd/pipeline/tree/v0.4.0/examples#examples)           |
-| [v0.3.1](https://github.com/tektoncd/pipeline/releases/tag/v0.3.1)           | [Docs @ v0.3.1](https://github.com/tektoncd/pipeline/tree/v0.3.1/docs#tekton-pipelines)           | [Examples @ v0.3.1](https://github.com/tektoncd/pipeline/tree/v0.3.1/examples#examples)           |
-| [v0.3.0](https://github.com/tektoncd/pipeline/releases/tag/v0.3.0)           | [Docs @ v0.3.0](https://github.com/tektoncd/pipeline/tree/v0.3.0/docs#tekton-pipelines)           | [Examples @ v0.3.0](https://github.com/tektoncd/pipeline/tree/v0.3.0/examples#examples)           |
-| [v0.2.0](https://github.com/tektoncd/pipeline/releases/tag/v0.2.0)           | [Docs @ v0.2.0](https://github.com/tektoncd/pipeline/tree/v0.2.0/docs#tekton-pipelines)           | [Examples @ v0.2.0](https://github.com/tektoncd/pipeline/tree/v0.2.0/examples#examples)           |
-| [v0.1.0](https://github.com/tektoncd/pipeline/releases/tag/v0.1.0)           | [Docs @ v0.1.0](https://github.com/tektoncd/pipeline/tree/v0.1.0/docs#tekton-pipelines)           | [Examples @ v0.1.0](https://github.com/tektoncd/pipeline/tree/v0.1.0/examples#examples)           |
-
-_See [our API compatibility policy](api_compatibility_policy.md) for info on the
-stability level of the API._
-
-_See [our Deprecations table](docs/deprecations.md) for features that have been
-deprecated and the earliest date they'll be removed._
+### Releases
+
+- See [our releases page](https://github.com/tektoncd/pipeline/releases) to find documentation
+for each release.
```

Comment:
This is slightly easier for the maintainers but an extra click or two for our users... not sure we should remove it?
For the out-of-date issue, we could automate it?

Reply:
I like the idea of automating it but don't think I have the bandwidth for that right now. I am not too concerned about the extra click or two but happy to close this PR if you'd like to keep the table around.

Reply:
I'd prefer to keep it around until we can automate it, yeah. Seems like something we could do in `hack/update-codegen.sh` maybe?

Reply:
Sounds good -- created #5247